apps/meteor/app/livechat/server/lib/Departments.ts

```diff
@@ -24,12 +25,7 @@ class DepartmentHelperClass {
 			throw new Error('error-failed-to-delete-department');
 		}
 
-		const agentsIds: string[] = await LivechatDepartmentAgents.findAgentsByDepartmentId<Pick<ILivechatDepartmentAgents, 'agentId'>>(
-			department._id,
-			{ projection: { agentId: 1 } },
-		)
-			.cursor.map((agent) => agent.agentId)
-			.toArray();
+		const removedAgents = await LivechatDepartmentAgents.findByDepartmentId(department._id, { projection: { agentId: 1 } }).toArray();
```

Comment:
this is a bug fix, since `LivechatDepartmentAgents.findAgentsByDepartmentId` is paginated and return only 50 items..